widgets/hits.js

```diff
@@ -3,9 +3,6 @@ var ReactDOM = require('react-dom');
 
 var utils = require('../lib/utils.js');
 
-var bindProps = require('../decorators/bindProps');
```

Comment:
> and much more clear

Ok so I will do this for all other widgets while I am testing them one after one